pages/a/decred-101.vue

```diff
@@ -236,9 +236,9 @@
               </BlockHint>
 
               <p>
-                Decred stakeholders lock their funds for an unknown period of time to
+                Decred stakeholders lock their funds for a pseudorandom period of time (average: ~28 days; minimum: 1 day; maximum: 142 days) to
```

Comment:
Good addition. I want to display only average, not to scare people off with 142 days
```suggestion
                Decred stakeholders lock their funds for a pseudorandom period of time (average: ~28 days) to
```